### ISSUE TITLE
Confusing use of "Validation Set" in beginner example

### ISSUE THREAD

**Issue by omatai** [tensorflow/tensorflow#38735]
Thank you for submitting a TensorFlow documentation issue. Per our GitHub
policy, we only address code/doc bugs, performance issues, feature requests, and
build/installation issues on GitHub.

The TensorFlow docs are open source! To get involved, read the documentation
contributor guide: https://www.tensorflow.org/community/contribute/docs

## URL(s) with the issue:

https://www.tensorflow.org/tutorials/quickstart/beginner

## Description of issue (what needs changing):

Tutorial makes use of model.evaluate(), and the documentation says that this is usually done on a "Validation Set". Everything else I read (glossary, docs for model.fit()... including validation set parameters) points to this relating to a "Test Set" since it occurs after the training phase, and the parameters passed are "x_test" and "y_test". The confusion is unhelpful to beginners. Change from "Validation Set" to "Test Set"?

### Correct links

n/a

### Parameters defined

n/a

### Returns defined

n/a

### Raises listed and defined

n/a

### Usage example

n/a

### Request visuals, if applicable
n/a

### Submit a pull request?

No. I'm a beginner, so I don't want to do anything, lest I create more confusion.

**Comment by oke-aditya** [comment]
Hi, @omatai let me help you clarify.
We usually perform training on data using `model.fit(x_train, y_train)`. Since we know this data we cannot perform testing on this data. Hence we create two sets.

- One is called the validation set and other called the test set. We validate our learning on the validation and performing testing on the test set. Here for simplicity, we did not divide our data into three parts.

- I think rather than skipping it maybe raise a PR which shows training, validation and testing all three done clearly.

Would that be fine @omatai ?

**Comment by omatai** [comment]
As a beginner quickstart example, it might help to say:

  *   Model.fit() has fit the model to the data using the training set, but we have not provided a validation set to model.fit() so that we can control the quality of fit. Lets bypass any concerns about over-fitting, and proceed to use model.evaluate() on a test set.

That is in line with my understanding of what the code is doing. I would keep the code the same, and adjust the documentation text.

**Comment by oke-aditya** [comment]
- Ok Let me fix as you say. I will make it clearer in documentation.

**Comment by oke-aditya** [comment]
@omatai I have fixed as you said. Please have a look at the pull request.

**Comment by oke-aditya** [comment]
@omatai This PR [link](https://github.com/tensorflow/docs/pull/1549) is added. I hope it solves the issue.

**Comment by oke-aditya** [comment]
- It is fixed. PR is merged.

**Comment by omatai** [comment]
I don’t think it is fixed in  a way that will prevent future confusion – the links to both Validation Set and Test Set describe different things: validation set is related to training phase; test set is related to testing phase. The comment relates to testing phase, so if validation set is going to be mentioned at all, it needs further explanation as to why… which could be that the standard naming convention used on Tensorflow is not strictly adhered to and not universal.

I don’t think I’m confused any more, but I expect others will be confused at some point. But hey: I’m reasonably rusty on all this… and have much higher priorities. Thanks for your time 😊

**Comment by gowthamkpr** [comment]
@omatai I think the fix solves the problem as the definition for validation and test set has been interlinked. Please go ahead and close this issue as I think there is no confusion here. Thanks!
